roles/ovirt_ovirt_collectd/ovirt_collectd_package/tasks/main.yml

```diff
@@ -2,14 +2,14 @@
 - name: Install collectd package
   yum:
     name: '{{ collectd_package_name }}'
-    state: latest
+    state: present
```

Comment:
Yes wanted to consult with you, maybe its better to add this error (403) to the skip list, since I don't want to change behavior. what do you think?

Reply:
Hmm maybe add something like this https://github.com/oVirt/ovirt-ansible-collection/blob/0153fdb3070fc6c96eefc5d48324c712041b7d73/automation/build.sh#L76-L77
or ignore the line but you would need to add the comments why we do the ignore.

Reply:
I added a file named .ansible-lint, with the content:
"
skip_list:
  - '701'
  - '208' 
"
you think it's better to add it in the build.yml ?
 
I ignored 208: complain about permissions not set but everything works properly
701 complained about No 'galaxy_info' found" but we do not use galaxy, I was not able to understand why I have to add galaxy_info and why

Reply:
Adding the 0644 is okay with me 
But still, see that the state is `latest` think it would be best to ignore the error and create different PR for this change because this changes the behaviour of the code

Reply:
@mnecas I consulted Martin, he said it will install the latest anyway with fresh install, and we shouldn't worry about upgrade since the upgrade. You think it's better to create a different pr?